wordpress-mobile/WordPress-Android
# Fix Stats Widget broken for API 19

This PR fixes #8262 by adding a simplified version of StatsWidget layout file that doesn't use Vector drawables or references to them in the `Style` attribute.

Changes are very trivial, and required since "app-comp" things are not loaded in Widgets.

To test:
- Start a device with API 19 and add the Stats Widget to home.
- It should display fine



Note: Merge #8263 before of this PR.
